### ISSUE TITLE
Missing Indexes

### ISSUE THREAD

**Issue by jefhar** [craigrodway/LoginPersist#14]
In `private function pruneLogins()`, you are querying by `created`. This field needs an index, especially if 4-6 million rows are queried on every page load.

In `destroyLogins()`, you are querying by `user`. This field needs an index. There exists a compound index, `user, series`, but that index is not used in this query.

**Comment by craigrodway** [comment]
Thanks @jefhar, it's absolutely right that those columns should have indexes.

Are you operating a site that has 4-6 million rows in the `login_persist` table? If so, have you added those indexes and do they improve things?

**Comment by jefhar** [comment]
@craigrodway Yes, we have seen improvements on out monitoring dashboard after manually adding the index.

**Comment by craigrodway** [comment]
Thanks @jefhar, I'll look to add those to the module.
